packages/@apphosting/adapter-angular/src/interface.ts

```diff
@@ -12,6 +12,12 @@ export interface OutputBundleOptions {
   needsServerGenerated: boolean;
 }
 
+export interface EnvironmentVariable {
+  variable: string;
+  value: string;
+  avalability: string[];
```

Comment:
string or enum?

Reply:
I think we should stay consistent with the publisher schema, so using string here: https://source.corp.google.com/piper///depot/google3/third_party/gcp_buildpacks/pkg/firebase/apphostingschema/apphostingschema.go;l=57;rcl=652323474

Reply:
I think the use cases are slightly different; the publisher uses strings because it's parsing user input which doesn't necessarily have the ability to leverage enums (since it's originally just yaml format).

Besides consistency, is there a reason to keep these as strings? I would argue that constraining these to enums may help us ensure that adapter authors don't accidentally pass us the wrong values.

See http://principles-wiki.net/principles:easy_to_use_and_hard_to_misuse#:~:text=A%20module%20is%20easy%20to,the%20compiler%20signals%20wrong%20usages.

Reply:
Hmm that makes sense, I guess besides consistency they needs to be merged in the publisher so thought it'd be nice to follow the same interface. But enums sounds more accurate in the adapter author case, changing it to enums.